src/Services/OData/ODataRequest.cs

```diff
@@ -16,53 +16,195 @@
 namespace SenseNet.Portal.OData
 {
     internal enum OutputFormat { None, JSON, VerboseJSON, Atom, Xml }
-    public enum InlineCount { None, AllPages }
-    public enum MetadataFormat {None, Minimal, Full}
 
+    /// <summary>
+    /// Defines values for handling collection count.
+    /// </summary>
+    public enum InlineCount
+    {
+        /// <summary>
+        /// Not defined value. This is the default.
+        /// </summary>
+        None,
+        /// <summary>
+        /// Defines that the client requires the total count of collection even if the
+        ///  request has restrictions in connection with cardinality (e.g. "$top=10").
+        /// </summary>
+        AllPages
+    }
+
+    /// <summary>
+    /// Defines values for metadata verbosity.
+    /// </summary>
+    public enum MetadataFormat
+    {
+        /// <summary>
+        /// There is no metadata writing at all.
+        /// </summary>
+        None,
+        /// <summary>
+        /// Writing metadata with minimal information.
+        /// </summary>
+        Minimal,
+        /// <summary>
+        /// Writint the whole metadata. This is the default value.

```

Comment:
Writing the whole metadata. This is the default value.

Reply:
ok